debezium/src/main/scala/akka/stream/alpakka/debezium/DebeziumRecord.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.debezium
+
+import org.apache.kafka.connect.data.{SchemaBuilder, Struct}
+import org.apache.kafka.connect.source.SourceRecord
+import org.apache.kafka.connect.storage.Converter
+
+private[debezium] trait DebeziumRecord {
+  private[debezium] val converter: Converter
+
+  val record: SourceRecord
+
+  lazy val payload: Array[Byte] = {
+    val schemaBuilder = SchemaBuilder.struct.field("key", record.keySchema)
+    if (record.valueSchema != null) schemaBuilder.field("value", record.valueSchema)
+
+    val schema = schemaBuilder.build
```

Comment:
Building a schema seems like something we only need to do once. Perhaps it could be triggered lazily and stored in `DebeziumSettings`, or in a companion object of `DebeziumRecord`.

Reply:
Actually, Debezium emits events for multiple tables and tables have evolution too, the schema can change at any time.

Reply:
Got it.  I guess we could cache schemas, but probably not something we need to consider for now.